### PR TITLE
Update sklmessages_ar_SA.properties

### PR DESCRIPTION
**Pull request by HHHGoBroooo**
Updated And fixed a lot of misspelling

### CODE REVIEW

File: assets/launcher/lang/sklmessages_ar_SA.properties [hunk 1]
```diff
@@ -66,91 +66,92 @@ alert.button.cancel=الغاء
 
 # About
 about.translation=تمت الترجمة من قبل:
-about.translation.authors=Mh Mazen
+about.translation.authors=Mh Mazen و Mahdi ßaghdadi
```
Comment (skmedix) [inline assets/launcher/lang/sklmessages_ar_SA.properties:69]:
```suggestion
about.translation.authors=Mh Mazen, Mahdi ßaghdadi
```

**Review by skmedix — CHANGES_REQUESTED** [review]

File: assets/launcher/lang/sklmessages_ar_SA.properties [hunk 2]
```diff
@@ -66,91 +66,92 @@ alert.button.cancel=الغاء
...
 # Login Screen
-login.button.type.microsoft=Login with Microsoft
-login.button.type.offline=Login offline
-login.label.username=Username
-login.dropdown.name=Logged In Users
-login.button.play=Play
-login.button.logout=Log Out
-login.button.switch.offline=Switch to offline mode
-login.button.switch.online=Switch to online mode
+login.button.type.microsoft=سجل ب microsoft
```
[the remark this answers is not in the document]
Comment (HHHGoBroooo) [reply assets/launcher/lang/sklmessages_ar_SA.properties:72]: ok

File: assets/launcher/lang/sklmessages_ar_SA.properties [hunk 3]
```diff
@@ -66,91 +66,92 @@ alert.button.cancel=الغاء
...
+manager.profile.menu.delete=احذف
+
+manager.editor.header.title=تعديل الملفات
+manager.editor.footer.button.save=حفظ
+manager.editor.option.name.name=اسم الملف
+manager.editor.option.name.placeholder=ملف غير مسمى
+manager.editor.option.version.name=لإصدار
+manager.editor.option.directory.name=موقع ملفات اللعبة
+manager.editor.option.directory.placeholder=استخدم اموقع الأصلي
+manager.editor.option.directory.info.label= بعض التعديلات أن يكون التثبيت موجودًا في مجلد minecraft. لكي تعمل
```
[the remark this answers is not in the document]
Comment (HHHGoBroooo) [reply assets/launcher/lang/sklmessages_ar_SA.properties:121]: github has this problem bc the arabic and english mix doesnt work perfectly bc arabic is a hard language to add perfectly

File: assets/launcher/lang/sklmessages_ar_SA.properties [hunk 4]
```diff
@@ -3,7 +3,7 @@ lang.code=ar_SA
 
 general.issues=تبليغ عن مشكلة
 general.support=دعم على Patreon
-general.discord=Join our Discord
+general.discord=انظم الي سيرفرنا في discord
```
[the remark this answers is not in the document]
Comment (HHHGoBroooo) [reply assets/launcher/lang/sklmessages_ar_SA.properties:6]: ok

File: assets/launcher/lang/sklmessages_ar_SA.properties (hunk 1 above)
Comment (HHHGoBroooo) [reply assets/launcher/lang/sklmessages_ar_SA.properties:69]: " و" 
means " and "

**Review by skmedix — APPROVED** [review]

**Pull request merged**